packages/synapse-react-client/src/components/DropdownSelect/DropdownSelect.tsx

```diff
@@ -88,7 +107,16 @@ export default function DropdownSelect(props: DropdownSelectProps) {
             }
           }}
         >
-          {options[selectedIndex ?? selectedIndexLocal]}
+          {buttonText ? (
+            <Typography
+              variant="label"
+              sx={{ lineHeight: '20px', fontSize: '16px' }}
+            >
+              {buttonText}
+            </Typography>
```

Comment:
What's wrong with the default button text style? Why would we only want to change that in this one place?

```suggestion
            buttonText
```

Reply:
Changing the style in the popover component where it's used instead.